src/mongo_types.ts

```diff
@@ -472,7 +472,10 @@ export class TypedEventEmitter<Events extends EventsDescription> extends EventEm
   }
 }
 
-/** @public */
+/**
+ * @public
+ * @deprecated Will be removed in favor of `AbortSignal` in the next major release.
```

Comment:
```suggestion
 * @deprecated Will be removed in the next major release.  This type is currently not used in our public API.
```

This type isn't used anywhere in our public API, is it?  If not, no need to specify a replacement

Reply:
Updated.